cfg/dist.js

```diff
@@ -1,42 +0,0 @@
-'use strict';
```

Comment:
How will you split execution between `local` (unoptimized, for development) and `dist`(production version) ?